packages/init/README.md

```diff
@@ -1,51 +0,0 @@
-<p align="center">
```

Comment:
no read me any more?

Reply:
We don't have `README.md`s in any of the new packages. We should since they will be published to npm.

`slice-machine-ui` should contain the main README since that is what developers will see in their `package.json`. We can link the root-level README to that README.

Reply:
I added READMEs to each package:

- https://github.com/prismicio/slice-machine/tree/dev-plugins/packages/init
- https://github.com/prismicio/slice-machine/tree/dev-plugins/packages/manager
- https://github.com/prismicio/slice-machine/tree/dev-plugins/packages/plugin-kit
- https://github.com/prismicio/slice-machine/tree/dev-plugins/packages/start-slicemachine

Reply:
Nice :)